core/commands.go

```diff
@@ -600,6 +610,7 @@ func init() {
 	diceCmds["MSET"] = msetCmdMeta
 	diceCmds["JSON.SET"] = jsonsetCmdMeta
 	diceCmds["JSON.GET"] = jsongetCmdMeta
+	diceCmds["JSON.TYPE"] = jsontypeCmdMeta
```

Comment:
(**Also please add this new object to the bottom of the list**) is it refer this?

Reply:
That comment can be ignored for now. Thanks for making the other changes!